apps/schematic/api/schematic_api/openapi/openapi.yaml

```diff
@@ -232,18 +232,169 @@ paths:
       tags:
       - Storage
       x-openapi-router-controller: schematic_api.controllers.storage_controller
-  /components/{componentLabel}/attributes:
+  /assetTypes/{assetType}/manifests/{manifestId}/json:
```

Comment:
Okay I think this answers a previous question, why do we need a whole endpoint for this?